src/components/Episodes/Episodes.jsx

```diff
@@ -9,7 +9,18 @@ import {
 
 export default function Episodes(){
 
-	const episodes = data.map(RENDER.episode);
+	const episodes = data.map( (obj, index) => {
+		//passing prev and next video src for video nav buttons
+		const prev = index !== 0 
+			? data[index-1] 
+			: {};
+		
+		const next = index !== data.length-1 
+			? data[index+1] 
+			: {};
+
+		return RENDER.episode(obj, index, prev, next);
```

Comment:
I think we should just give `RENDER.episode` `data[index-1]` and `data[index+1]`  and then the logic relating to handling whether or not next/prev is `undefined` is just handled in the `<Episode>` component, or whatever.

Aside from that, this sorta logic should be part of the `RENDER.episode` util, as it's part of the episode stuff, not the episodes render itself, if that makes sense?